packages/sycamore/src/generic_node/dom_node.rs

```diff
@@ -289,6 +289,34 @@ pub fn render(template: impl FnOnce() -> View<DomNode>) {
     render_to(template, &document.body().unwrap_throw());
 }
 
+/// Render a [`View`] under a `parent` node, in a way that can be cleaned up.
+/// This function is intended to be used for injecting an ephemeral sycamore view into a non-sycamore app
+/// (for example, a file upload modal where you want to cancel the upload if the modal is closed).
+/// You should only use this function while refactoring your app, and you should aim to have a single
+/// call to [`render`] or [`render_to`] at the top level of your app long-term.
+/// For rendering a view that will never be unmounted from the dom, use [`render_to`] instead.
+/// For rendering under the `<body>` tag, use [`render`] instead.
+///
+/// _This API requires the following crate features to be activated: `dom`_
+#[must_use = "please hold onto the ReactiveScope until you want to clean things up, or use render_to() instead"]
+pub fn render_temporarily_to(
+    template: impl FnOnce() -> View<DomNode>,
+    parent: &Node,
+) -> ReactiveScope {
+    create_root(|| {
+        insert(
+            &DomNode {
+                id: Default::default(),
+                node: parent.clone(),
+            },
+            template(),
+            None,
+            None,
+            false,
+        );
+    })
+}
```

Comment:
All this is duplicated in `render_to`. Maybe replace `render_to` with a call to this function.